docker-compose.yml

```diff
@@ -1,17 +1,19 @@
 version: '2'
 services: 
-    xnat-web:
-       build: ./tomcat
+   xnat-web:
+       build: ./xnat17
        ports:
-        - "8081:8080"
+         - "8081:8080"
        volumes:
-         - ./webapps:/opt/tomcat/webapps
-         - ./plugins:/data/xnat/home/plugins
```

Comment:
Add back this line. That's how we can get plugins into XNAT.

Reply:
This is already exposed trough the volume /data/xnat.

Reply:
:+1: Good point. Consider this comment rescinded.